app/helpers/samples_helper.rb

```diff
@@ -41,29 +41,25 @@ def controlled_vocab_list_form_field(sample_controlled_vocab, element_name, valu
     controlled_vocab_form_field(sample_controlled_vocab, element_name, values, nil)
   end
 
-  def linked_custom_metadata_form_field(attribute,resource,element_name, element_class,depth)
-    linked_cms = resource.linked_custom_metadatas.select{|cm|cm.custom_metadata_attribute==attribute}
-
-    id = linked_cms.blank? ? nil : linked_cms.select{|cm| cm.custom_metadata_type.id == attribute.linked_custom_metadata_type.id}.first.id
+  def linked_custom_metadata_multi_form_field(attribute, value, element_name, element_class)
+    render :partial => 'custom_metadata/fancy_linked_custom_metadata_multi_attribute_fields',
+           :locals=> {:value=>value, :attribute => attribute,:element_name => element_name,:element_class => element_class, :collapsed=>false}
+  end
```

Comment:
use `partial:` as the keys, rather than the older fashioned `:partial=>` (applies to the other keys too, not just partial)